tests/test_parse_fields.py

```diff
@@ -6,25 +6,38 @@
 test_fields = ["body:text", "number_of_pages:int", "book_url:url"]
 
 all_raw_fields = [
-    "auto:auto",
-    "bool:bool",
-    "char:char",
-    "date:date",
-    "datetime:datetime",
-    "email:email",
-    "float:float",
-    "int:int",
-    "slug:slug",
-    "text:text",
-    "time:time",
-    "url:url",
-    "uuid:uuid",
+    "auto : auto",
+    "bigauto : bigauto",
+    "bigint : bigint",
+    "binary : binary",
+    "bool : bool",
+    "char : char",
+    "date : date",
+    "datetime : datetime",
+    "decimal : decimal",
+    "duration : duration",
+    "email : email",
+    "file : file",
+    "filepath : filepath",
+    "float : float",
+    "genericip : genericip",
+    "image : image",
+    "int : int",
+    "nullbool : nullbool",
+    "positiveint : positiveint",
+    "positivesmallint : positivesmallint",
+    "slug : slug",
+    "smallint : smallint",
+    "text : text",
+    "time : time",
+    "url : url",
+    "uuid : uuid",
```

Comment:
There's no need to add spaces between `:` which in turn adds more stuff into the parsing logic.